### ISSUE TITLE
[sumologic_s3_source] How to set "automatic" scan interval and S3 Region?

### ISSUE THREAD

**Issue by yermulnik** [SumoLogic/terraform-provider-sumologic#231]
- [Sumo Logic Docu](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Amazon-Web-Services/AWS-S3-Source#Scan+Interval+in+step+11) suggests to set `Scan Interval` to `Automatic` when used along with `SNS Subscription Endpoint` but `scan_interval` from `sumologic_s3_source` resource is of type `int` — how to set it to `automatic`?

- `sumologic_s3_source` resource does not provide `region` attribute despite this being available via UI — how to provide this value via Terraform?
![image](https://user-images.githubusercontent.com/1274789/125112991-a7632e80-e0f0-11eb-9da4-d17cdf96f511.png)

**Comment by yermulnik** [comment]
Sp to set `Scan Interval` to `Automatic` one needs to set its value to `-1` per https://help.sumologic.com/03Send-Data/Sources/03Use-JSON-to-Configure-Sources/JSON-Parameters-for-Hosted-Sources#aws-log-sources (kudos to TJ Vasques Jr).

**Comment by frankreno** [comment]
Yes you need to set that to -1, we will update docs good call out.

As for the region, there is an inconsistency it seems in the API and UI.  Any sources created via API or UI will also use `Others` region and it is not yet possible to to choose other regions.
